src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -262,19 +262,25 @@ impl CatalogState {
         let cluster = &self.clusters_by_id[&id];
         let row = self.pack_privilege_array_row(cluster.privileges());
         let privileges = row.unpack_first();
-        let (size, disk, replication_factor, azs) = match &cluster.config.variant {
-            ClusterVariant::Managed(config) => (
-                Some(config.size.as_str()),
-                Some(config.disk),
-                Some(config.replication_factor),
-                if config.availability_zones.is_empty() {
-                    None
-                } else {
-                    Some(config.availability_zones.clone())
-                },
-            ),
-            ClusterVariant::Unmanaged => (None, None, None, None),
-        };
+        let (size, disk, replication_factor, azs, introspection_debug, introspection_interval) =
+            match &cluster.config.variant {
+                ClusterVariant::Managed(config) => (
+                    Some(config.size.as_str()),
+                    Some(config.disk),
+                    Some(config.replication_factor),
+                    if config.availability_zones.is_empty() {
+                        None
+                    } else {
+                        Some(config.availability_zones.clone())
+                    },
+                    Some(config.logging.log_logging),
+                    config.logging.interval.map(|d| {
+                        Interval::from_duration(&d)
+                            .expect("planning ensured this convertible back to interval")
```

Comment:
I'm not sure about this, I stole it from the cluster_schedules table...

Reply:
Yeah, this is fine. We create the `Duration` from an `Interval` in planning, so converting it back should always succeed.